<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>215</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>215</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>username</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="repeatPasswordLabel">
      <property name="text">
       <string>repeat password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="repeatPasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>Login</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="registerLabl">
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="registerButton">
        <property name="text">
         <string>Register</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>repeatPasswordLineEdit</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
